src/component/Navbar.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link, NavLink } from 'react-router-dom';
+import '../index.css';
+
+const Navbar = ({ title, routes }) => (
+  <nav className="nav-bar">
+    <Link className="nav-brand" to="/">{title}</Link>
```

Comment:
- [ ] The Logo on zeplin doesnt have an underline while your has. Kindly modify yours so that it matches Zeplin design

your design 👇🏽 
![image](https://github.com/theresetuyi/React-Bookstore/assets/63473584/192a3b8e-0cdf-4133-b094-56f1044f8fd6)

Zeplin design 👇🏽 
![image](https://github.com/theresetuyi/React-Bookstore/assets/63473584/d98180b4-4858-4689-a126-2345116a757c)